lib/perl/OVH/Bastion/allowkeeper.inc

```diff
@@ -1014,4 +1014,21 @@ sub is_valid_ttl {
     return R('KO_INVALID_PARAMETER', msg => "Invalid TTL ($ttl), expected an amount of seconds, or a duration string such as '2d8h15m'");
 }
 
+# used by groupList and accountList
+sub build_re_from_wildcards {
+    my %params    = @_;
+    my $wildcards = $params{'wildcards'};
+
+    # to avoid modifying the caller's array
+    my @relist = @$wildcards;
+
+    for (@relist) {
+        $_ = quotemeta;
+        s/\\\*/.*/g;
+        s/\\\?/./g;
+        $_ = '^' . $_ . '$';
+    }
+    return R('OK', value => join("|", @relist));
```

Comment:
You might return a `qr//` here and use it as `=~ $re` instead of `=~ /$re/`, no need to recompile it in every check loop turn

Reply:
done